src/main/java/com/conveyal/gtfs/model/RouteNetwork.java

```diff
@@ -69,7 +70,7 @@ public Writer(GTFSFeed feed) {
 
         @Override
         public void writeHeaders() throws IOException {
-            writer.writeRecord(new String[] {NETWORK_ID_NAME, ROUTE_ID_NAME});
+            writer.writeRecord(new String[] {NETWORK_ID_FIELD, ROUTE_ID_FIELD});
```

Comment:
Reuse the headers from `packRouteNetworks`, or alternately, reuse this method in `packRouteNetworks` (if you do, reuse the corresponding method in `packStopAreas`). Also check that the order of the columns is the same as in the content rows. (Is this code actually being called?).

Reply:
Code not being called, so I have removed it. Same with stop areas.